src/plugins/workspace/public/plugin.test.ts

```diff
@@ -40,7 +40,7 @@ describe('Workspace plugin', () => {
       savedObjectsManagement: savedObjectManagementSetupMock,
       management: managementPluginMock.createSetupContract(),
     });
-    expect(setupMock.application.register).toBeCalledTimes(5);
+    expect(setupMock.application.register).toBeCalledTimes(6);
```

Comment:
Do you, by chance, have an explanation for this number? It would help to have a comment... unless it is obvious.

Reply:
Sure, I have added a annotate.